compiler/damlc/daml-prim-src/GHC/Num.daml

```diff
@@ -120,22 +120,16 @@ instance Signed Int where
     signum x = if x == 0 then 0 else if x <= 0 then (-1) else 1
     abs x = if x <= 0 then negate x else x
 
-#ifdef DAML_NUMERIC
-#define IF_NUMERIC(a,b) a
 #define NUMERIC_LIT(n) (primitive @"BECastNumeric" ((n) : Numeric 10))
-#else
-#define IF_NUMERIC(a,b) b
-#define NUMERIC_LIT(n) n
-#endif
 
-instance Additive IF_NUMERIC((Numeric n), Decimal) where
-    (+) = primitive @IF_NUMERIC("BEAddNumeric", "BEAddDecimal")
```

Comment:
can we get rid of `BTDecimal` now? Also the primitives `BEAddDecimal` et al.?

Reply:
good point, I think we can but it looks like it’s a larger change so I’ll move it to a separate PR.

Reply:
that sounds good, thanks!